artemis-docker/readme.md

```diff
@@ -134,7 +148,7 @@ From within the `$ARTEMIS_DIST` folder:
 $ docker build -f ./docker/Dockerfile-ubuntu-11-jre -t artemis-ubuntu .
 ```
 
-# For Ubuntu (Build for linux ARMv7/ARM64)
+## For Ubuntu (Build for linux ARMv7/ARM64)
 ```
 $ docker buildx build --platform linux/arm64,linux/arm/v7 --push -t {your-repository}/apache-artemis:2.17.0-SNAPSHOT -f ./docker/Dockerfile-ubuntu-11 .
```

Comment:
Were you able to make this work with 2.30?

Reply:
Hi Clebert. Sorry I missed the commit msg req, and thanks Justin for fixing it.
Yes I was able to get a cross platform version of 2.30.0 built. I updated karaf to 4.4.3 as per main which allowed the local build to complete successfully. The other steps worked as written in the doc.